packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -110,26 +110,26 @@ const getLowerCaseNames = (room, nameDefault = '') => {
 	};
 };
 
-// RocketChat.Notifications['onUser']('rooms-changed', );
-
-const mergeSubRoom = (record/*, t*/) => {
-	const room = Tracker.nonreactive(() => RocketChat.models.Rooms.findOne({ _id: record.rid }));
-	if (!room) {
-		return record;
-	}
-	record.lastMessage = room.lastMessage;
-	record.lm = room._updatedAt;
-	return _.extend(record, getLowerCaseNames(record));
+const mergeSubRoom = subscription => {
+	const room = RocketChat.models.Rooms.findOne(subscription.rid) || { _updatedAt: subscription.ts };
+	subscription.lastMessage = room.lastMessage;
+	subscription.lm = room._updatedAt;
+	return Object.assign(subscription, getLowerCaseNames(subscription));
 };
 
-RocketChat.callbacks.add('cachedCollection-received-rooms', (room) => {
+const mergeRoomSub = room => {
 	const sub = RocketChat.models.Subscriptions.findOne({ rid: room._id });
 	if (!sub) {
-		return;
+		return room;
 	}
 	const $set = {lastMessage : room.lastMessage, lm: room._updatedAt, ...getLowerCaseNames(room, sub.name)};
```

Comment:
Please break this object in multiple lines and remove the space before the first colon.